### DISCUSSION TITLE
installation failed

### DISCUSSION THREAD

**Discussion by pzafar1974** [Security-Onion-Solutions/securityonion#2739] [General]
Hi 

I am trying to upgrade the So from 16.04 to SO 2.3.21. When I run so script it failed in the middle. Based of the search it seems it can not install the docker images. I tried to run so-docker-refresh, I am getting the same error here too. Please see the screenshot.

I would appreciate the help.

Thanks
[
![Screen Shot 2021-01-26 at 9 22 55 PM](https://user-images.githubusercontent.com/16450618/105880917-52899b00-601d-11eb-8938-af59fa35b749.png)
](url)

**Comment by TOoSmOotH** [comment]
I assume you already did the update to Ubuntu 18.04?

**Reply by pzafar1974** [reply]
Yes, I did already. Following documentations.

**Reply by pzafar1974** [reply]
Problem solved by itself not sure what was causing this. Does anyone have any clue?
